docs/user/01-10-access-nats-server.md

```diff
@@ -0,0 +1,36 @@
+# Accessing the NATS Server Using CLI
+
```

Comment:
If this goes to DITA, we're going to need a shortdesc anyway; so we could use this to state why the reader would want to access the NATS server:

```suggestion
Interact directly with your NATS server deployment in Kyma using the NATS command-line interface (CLI). Use the CLI to inspect server status, manage streams and consumers, and troubleshoot message flow.
```